src/main/resources/templates/mail/invitation_approval_execute_submission_text.txt

```diff
@@ -0,0 +1,8 @@
+Dear [(${userFirstName})],
```

Comment:
Cannot we deal w/o the duplication? Smth like: build html template and clean it up from the tags? Redundancy will cause anomalies earlier or later